src/main/java/com/google/sps/data/DistancePair.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.lang.Comparable;
+
+/** A class representing a pair containing recommendation ID and distance */
+public final class DistancePair implements Comparable<DistancePair>{
```

Comment:
nit: maybe the name of this class needs to be a little clearer? Just personal opinion though so feel free to leave it if you think it's clear enough!

Reply:
I thought what to name it, but I can only think of this. Do you have any suggestion?

Reply:
It's okay leaving it as this - it wasn't a big problem :)